composer.json

```
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.4",
```

I got here through an issue that stated this was going to upgrade to php ^8. 
```suggestion
        "php": "^8",
```